### PR TITLE
Allow self-signed certs for S3 backend

### PR DESCRIPTION
**Pull request by MrCreosote**
Next:
Test with Minio with a SSC from feature branch (w/ @kkellerlbl 's help)

### CODE REVIEW

**Comment by MrCreosote** [comment]
Docker build fail is expected

File: docsource/releasenotes.rst [hunk 1]
```diff
@@ -1,6 +1,13 @@
 Workspace service release notes
 ===============================
 
+VERSION: 0.11.4 (Released TBD)
+----------------------------------
+
+UPDATES:
+
+* Added an option to trust all SLL certificates to the S3 backend (default false).
```
Comment (kkellerlbl) [inline docsource/releasenotes.rst:9]: * SSL

File: src/us/kbase/workspace/test/database/mongo/S3BlobStoreIntegrationTest.java [hunk 2]
```diff
@@ -46,6 +46,9 @@
 	// some reason.
 	
 	private static S3BlobStore s3bs;
+	// use to exercise the cert trusting code, although doesn't actually test against self
```
Comment (kkellerlbl) [inline src/us/kbase/workspace/test/database/mongo/S3BlobStoreIntegrationTest.java:49]: Do you want to add code to test against a Minio with a self-signed cert, or do you want to test in one of our deploy environments instead?

**Review by kkellerlbl — COMMENTED** [review]
Initial review while still draft.

File: docsource/releasenotes.rst (hunk 1 above)
Comment (MrCreosote) [reply docsource/releasenotes.rst]: Fixed

File: src/us/kbase/workspace/test/database/mongo/S3BlobStoreIntegrationTest.java (hunk 2 above)
Comment (MrCreosote) [reply src/us/kbase/workspace/test/database/mongo/S3BlobStoreIntegrationTest.java:49]: I'll test manually locally, and then we can test in CI. Running automated tests seems like a bit much

**Review by kkellerlbl — APPROVED** [review]
Looks good to me.

**Pull request merged**